tour/beerDEcoded.yaml

```diff
@@ -0,0 +1,188 @@
+id: beerDEcoded
+name: BeerDEcoded - StreetScienceCommunity
+description: BeerDEcoded - StreetScienceCommunity
+title_default: <b>BeerDEcoded</b>
+steps:
+  - title: <b>BeerDEcoded - StreetScienceCommunity</b>
+    content: >-
```

Comment:
I suggest this introduction for the tour:

"Beer contains DNA that comes from its ingredients (hops, malts, yeast) and some hundred microbes. There are 1,000+ yeasts used for brewing and 200+ hop varieties, each one bearing a different DNA and contributing to differentiate its properties.

Thanks for new technologies of sequencing it is easier nowadays to explore the potential of genome sequencing to understand the contribution of various species to product characteristics. The sequencing of the full genome of 157 brewing yeast strains was, for example, recently reported (<a href="https://pubmed.ncbi.nlm.nih.gov/27610566/">Gallone B, Steensels J, Prahl T, et al. 2016</a>).

Based on the identification of strains present in beer with desired characteristics, controlled experiments in which the microbial composition of the brew is altered could allow us to investigate if the presence of specific microorganisms affects flavour. The origin of each yeast species can be investigated; i.e. whether they come with the ingredients or from the environment at the production site. Furthermore, plant DNA, such as malt and hop varieties, can be found in beer samples, and the bacterial diversity can be mapped.
"

Reply:
In general: sounds very good. Maybe break it up into two boxes, so it is not too much text in one. 
Some hints:
- loos the (hops, malts, yeast). In our analysis we only find yeast. Since hops and malts are used early in the brewing process it is very likely that their cells were lysed. The sequencing protocol is not detecting them. To avoid confusion I would delete the (...)
- The sentence 'Thanks for new technologies...' is not clear to me. Maybe say it in easier words: Tanks to sequencing it is now easyer to find the different genomes contained in a beer sample and investigate their characteristics.
- The paper is very interesting.  :-)

Reply:
> In general: sounds very good. Maybe break it up into two boxes, so it is not too much text in one.
> Some hints:
> 
> * loos the (hops, malts, yeast). In our analysis we only find yeast. Since hops and malts are used early in the brewing process it is very likely that their cells were lysed. The sequencing protocol is not detecting them. To avoid confusion I would delete the (...)
> * The sentence 'Thanks for new technologies...' is not clear to me. Maybe say it in easier words: Tanks to sequencing it is now easyer to find the different genomes contained in a beer sample and investigate their characteristics.
> * The paper is very interesting.  :-)

What if we add also information about 3 steps of the process:
"The beerDeCoded process contains 3 consistent steps. The first step is DNA extraction from beer. Then, this DNA can be sequenced. That means that we can obtain the sequence of nucleotides for this specific DNA. Finally, we have to analyze received data in order to know which organisms this DNA is from." And add the process diagram from https://streetscience.community/projects/beerdecoded/
![image](https://user-images.githubusercontent.com/55543056/128480030-74cf7022-a108-4385-92c9-58e4dbfeb0e8.png)
And point out that we are currently on the third step?
Or this would be redundant information?

Reply:
That is a good idea. 
In general: It would be good if each box of the tour gets a title, explaining the content of the box. Like this, it will be easy for us to see the flow of the tour.
To this box: It would be good to connect it to the explanation before. E.g. To study the microbiome of beer you need to find out what is inside the beer. Getting this inside can be found by extracting the DNAs of the living organisms (yeast) inside the beer. Now you would also like to 'read' this DNA. This can be achieved by sequencing the DNA. 
Having this sequences now enabels us to do a Data analysis, which we will do in the following.

Reply:
done